### ISSUE TITLE
Can we use UniChem to get cross-references between KEGG and CHEBI?

### ISSUE THREAD

**Issue by saramsey** [RTXteam/RTX#1354]
See #1327

**Comment by kvarforl** [comment]
@saramsey Before I look too far into this, can you confirm that this is something we want, even after we remove the KEGG nodes from KG2?
(see Erica's comment [here](https://github.com/RTXteam/RTX/issues/1327#issuecomment-809577391))

I guess I'm a little bit confused at what to connect / how to cross reference if we don't have KEGG nodes in the graph. clarification is much appreciated 😁

**Comment by saramsey** [comment]
Good point, I had not considered that. However, I think even after we stop importing KG1, we should continue to have a bunch of KEGG nodes via CHEBI, MONDO, etc. We will likely lose *some* KEGG nodes since we don't have a master KEGG download to ETL (that requires a $$$ subscription). I don't think it will be feasible to just drop all KEGG cross-references; too many people use KEGG identifiers for various purposes, even though the KEGG database can't be downloaded without a subscription. We might be able to get information about KEGG compounds via the CTD database; will have to ask @zheng-liu.

**Comment by ericawood** [comment]
Based on the information [here](http://ctdbase.org/downloads/), it appears there are only cross references to KEGG pathways, not KEGG compounds.

**Comment by saramsey** [comment]
Thank you, @ericawood.

I guess my feeling is that we should try to retire KG1 if we can. At some point I might be persuaded to consider adding a KG2 build module that uses the KEGG API, but I'd like to exhaust all other options before resorting to that.

**Comment by saramsey** [comment]
@ericawood wrote:

>there are also cross references available between DrugCentral and HMDB (a fair amount for HMDB). I've updated extract-unichem.sh to make cross references between all of them (haven't pushed the commit yet though). Do you think that is a good update? (I can still back it out if desired)

good catch, @ericawood !  Yes, that seems like it would be great to add to KG2.

**Comment by ericawood** [comment]
This wasn't quite fixed in `KG2.6.1` due to a few typos:
https://github.com/RTXteam/RTX/blob/6f02f153f50488bd77119fdb01c10b6b325341fb/code/kg2/extract-unichem.sh#L44-L46

(it's missing the tabs)
